sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -36,11 +36,11 @@
     static DefaultServiceSpec from(
             RawServiceSpec rawSvcSpec, YAMLServiceSpecFactory.FileReader fileReader) throws Exception {
         RawScheduler rawScheduler = rawSvcSpec.getScheduler();
-
         String role = null;
         String principal = null;
         Integer apiPort = null;
         String zookeeper = null;
+        Boolean defaultGpuPolicy = false;
```

Comment:
As mentioned earlier, this is duplicate from the DefaultServiceSpec class. And hence it makes sense to pull this out as a constant. You can maybe place this inside DCOSConstants OR Capabilities class.